src/index.ts

```diff
@@ -10,7 +11,12 @@ const server = new ApolloServer({
 
 const port = 3001
 
-server.listen({ port }).then(({ url }) => {
-    // eslint-disable-next-line no-console
-    console.log(`🚀  Server ready at ${url}`)
-})
+function startServer() {
+    return startStandaloneServer(server, {
+        listen: { port: port }
+    })
+}
+
+// eslint-disable-next-line no-console
+startServer().then(() => console.log(`🚀  Server ready at: http://localhost:${port}`))
+
```

Comment:
Better! Here's another alternative using await. Let me know which you think looks better, they both work.

```suggestion
async function startServer(port = 3001) {
    await startStandaloneServer(server, {
        listen: { port: port }
    })

    // eslint-disable-next-line no-console
    console.log(`🚀  Server ready at: http://localhost:${port}`)
}

startServer()
```

Reply:
Also what do you think about renaming this file `main.ts`? `index` has such strong associations with frontend HTML `/` routes.

Reply:
this looks good to me.  I noticed you didn't await `startServer()` which I think is the right way to do it 👍

Reply:
index is also pretty common in nodejs apps for reference. I don't have a preference either way, just as long as it's easy to understand what is the entry point file

Reply:
We can't await at the top level in our version of ecmascript